src/nvim/os/unix_defs.h

```diff
@@ -19,6 +19,10 @@
 #define SPECIAL_WILDCHAR "`'{"
 
 // Unix system-dependent file names
+
+#define NVIM_CONF_DIR "$XDG_CONFIG_HOME/nvim/"
+#define NVIM_DATA_DIR "$XDG_DATA_HOME/nvim/"
```

Comment:
`//` does not look clean. I would avoid it when avoiding it costs nothing: just remove trailing slash here because in other places you already use leading slash.